allennlp/tests/fixtures/biaffine_dependency_parser_multilang/experiment.json

```diff
@@ -0,0 +1,98 @@
+{
+    "dataset_reader": {
+        "type": "universal_dependencies_multilang",
+        "languages": ["es", "it", "fr"],
+        "alternate": false,
+        "instances_per_file": 32,
+        "is_first_pass_for_vocab": true,
+        "token_indexers": {
+            "elmo": {
+                "type": "elmo_characters"
+            }
+        },
+        "use_language_specific_pos": false
+    },
+    "iterator": {
+        "type": "same_language",
+        "batch_size": 32,
+        "sorting_keys": [["words", "num_tokens"]],
+        "instances_per_epoch": 32
+    },
+    "model": {
+        "type": "biaffine_parser_multilang",
+        "arc_representation_dim": 10,
+        "dropout": 0.33,
+        "encoder": {
+            "type": "lstm",
+            "bidirectional": true,
+            "dropout": 0.33,
```

Comment:
Use smaller dimensions (all <10) and no dropout for test fixtures like this.  If you've already got artifacts created that depend on these dimensions, fine, as long as the test is very fast to run.